config/v1/feature_gates.go

```diff
@@ -83,6 +83,16 @@ var (
 		OwningProduct:       ocpSpecific,
 	}
 
+	FeatureGateExternalCloudProviderExternal = FeatureGateName("ExternalCloudProviderExternal")
```

Comment:
Is this definitely what we want to call this? Seems funny to have `External` twice, not sure it's super important what it's called since we intend to remove it eventually

Reply:
I agree it's not ideal, I kept the same naming scheme as `FeatureGateExternalCloudProviderGCP` and `FeatureGateExternalCloudProviderAzure` to stay consistent. Otherwise, something like `FeatureGateCloudControllerState` ?

Reply:
agree that the stutter isn't great, but since it follows the others i think it can be acceptable